x-pack/test_serverless/functional/page_objects/index.ts

```diff
@@ -47,4 +56,13 @@ export const pageObjects = {
   svlSearchCreateIndexPage: SvlSearchCreateIndexPageProvider,
   svlSearchInferenceManagementPage: SvlSearchInferenceManagementPageProvider,
   svlDataUsagePage: SvlDataUsagePageProvider,
+  assetDetails: AssetDetailsProvider,
+  observabilityLogsExplorer: ObservabilityLogsExplorerPageObject,
+  datasetQuality: DatasetQualityPageObject,
+  infraHome: InfraHomePageProvider,
+  infraHostsView: InfraHostsViewProvider,
+  infraLogs: InfraLogsPageProvider,
+  searchSynonyms: SearchSynonymsPageProvider,
+  searchQueryRules: SearchQueryRulesPageProvider,
+  searchPlayground: SearchPlaygroundPageProvider,
```

Comment:
solutions Page Objects were copied as a workaround while we need to relocate svl tests.